.github/workflows/juntagrico-ci.yml

```diff
@@ -27,6 +26,10 @@ jobs:
     - name: install dependencies
       run: |
         pip install --upgrade -r requirements-local.txt
```

Comment:
We could potentially add the --root-user-action=ignore switch to the pip command, as there were warnings that pip is used as a root user when I ran this workflow on my machine. However, it seems normal to run pip as the root user in this context.